server/src/main/java/org/opensearch/common/settings/IndexScopedSettings.java

```diff
@@ -222,7 +222,9 @@ public final class IndexScopedSettings extends AbstractScopedSettings {
         FeatureFlags.REPLICATION_TYPE,
         IndexMetadata.INDEX_REPLICATION_TYPE_SETTING,
         FeatureFlags.REMOTE_STORE,
-        IndexMetadata.INDEX_REMOTE_STORE_ENABLED_SETTING
+        IndexMetadata.INDEX_REMOTE_STORE_ENABLED_SETTING,
+        FeatureFlags.REMOTE_TRANSLOG_STORE,
```

Comment:
Type of `FEATURE_FLAGGED_INDEX_SETTINGS` may need to be changed to `Map<String, List<Setting>>`